Scripts/package.py

```diff
@@ -124,12 +124,19 @@ def build(self):
             shutil.rmtree(self.buildDirectory)
         run(["make", "--quiet", "clean"])
         run(["make", "--quiet", "runtime"])
-        run(["dotnet", "publish", path.join(SOURCE_DIRECTORY, "Dafny.sln"),
+        run(["dotnet", "build", path.join(SOURCE_DIRECTORY, "Dafny-RunCoco.sln"),
```

Comment:
Have you tried publishing both DafnyDriver.csproj and DafnyServer.csproj instead of Dafny.sln? That seemed to be another solution that doesn't require extra sln files.

Reply:
Good idea. I undid my changes, and accepted and merged your PR #1039.